manim/mobject/coordinate_systems.py

```diff
@@ -2185,9 +2185,10 @@ def __init__(
 
         z_axis.rotate_about_zero(-PI / 2, UP)
         z_axis.rotate_about_zero(angle_of_vector(self.z_normal))
+        z_axis.shift(-z_axis.number_to_point(self._origin_shift(z_axis.x_range)))
         z_axis.shift(
             self.x_axis.number_to_point(
-                self._origin_shift([z_axis.x_min, z_axis.x_max]),
+                self._origin_shift(self.x_axis.x_range),
```

Comment:
With the way log scaling works this would produce unintended effects for LogBase graphs. The `.x_min, x_max` approach is required.

Reply:
Ran an example of `ThreeDAxes` with the `x` and `y` axes as LogBase starting at 10^1 and saw that the axes did not align, as you stated.  After making the change you suggest, the LogBase axes do align.  I also added tests for LogBase, to ensure we don't have this issue again.